ddl/ddl_worker.go

```diff
@@ -122,42 +132,58 @@ func (d *ddl) checkOwner(t *meta.Meta) (*model.Owner, error) {
 	// the owner will update its owner status every 2 * lease time, so here we use
 	// 4 * lease to check its timeout.
 	maxTimeout := int64(4 * d.lease)
+	if flag == bgJobFlag {
+		// backgroun job doesn't need to guarantee other servers update the schema.
```

Comment:
@siddontang What is your opinion of `maxTimeout` value here? Seems `2 * d.lease` is too long.